WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
But what are you trying to achieve? Make pilgrimages more frequent than in vanilla? That doesn't fit for the mod, they were more rare.